Documentation/asagi.rst

```diff
@@ -81,42 +100,32 @@ followed by running ``fix_submodules`` to clone and set up the submodules.
 building SeisSol with ASAGI support
 -----------------------------------
 
-Simply add the following lines to the scons parameter file and make sure
-you use the same python and gcc package as for the compilation with
-ASAGI.
-
-.. code-block:: bash
-
-   asagi=yes
-   zlibDir=<path_to_ASAGI>/build/lib/
-
-**Known issues:** “can not find Asagi” while compiling SeisSol
+Simply turn on the option ``ASAGI=ON`` in the using ccmake.
 
-There are a couple of options that can be checked:
+generating the NetCDF input file
+--------------------------------
 
--  Is SeisSol compiled with a different python package?
--  Are the paths to ASAGI correctly included? Check
-   ``echo $PKG_CONFIG_PATH`` and ``echo $LD_LIBRARY_PATH``
--  When re-installing ASAGI again it might also help to remove the
-   temporary files .sconf_temp/ and .sconsign.dblite within the SeisSol
-   folder
+using python 
+~~~~~~~~~~~~~~~
 
-generating the netcdf input file
---------------------------------
+The most straightforward way to generate ASAGI file is to use the netCDF4 module of python.
+A typical example which generates a 2D ASAGI file can be found 
+`here <https://github.com/SeisSol/SeisSol/tree/master/preprocessing/science/generating_ASAGI_file.py>`__.
 
 using asagiconv
 ~~~~~~~~~~~~~~~
 
 Asagiconv (Located
 `here <https://github.com/SeisSol/SeisSol/tree/master/preprocessing/science/asagiconv>`__)
-allow querying data, visualising and exporting to netcdf data from the
+allow querying data, vizualising and exporting to NetCDF data from the
```

Comment:
typo

Reply:
> LGTM, but should we add a comment somewhere in the docu that you need to install the xdmf reader via pip?

I've added a comment in the fault output section. Ideally, there would be a more general output page, where I would have added that. But I don't want to do this now. Anyway, the idea is that the user can now search for `seissolxdmf` in the readthedocs and find the link.